src/content/RepoPage/RepoPage.js

```diff
@@ -59,11 +121,61 @@ const rows = [
 ];
 
 const RepoPage = () => {
+  const [totalItems, setTotalItems] = useState(0);
+  const [firstRowIndex, setFirstRowIndex] = useState(0);
+  const [currentPageSize, setCurrentPageSize] = useState(10);
   return (
     <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
       <div className="bx--row repo-page__r1">
         <div className="bx--col-lg-16">
           <RepoTable headers={headers} rows={rows} />
```

Comment:
The `<RepoPage>` component should have been removed. You'e still rendering it: https://deploy-preview-2251--carbon-tutorial.netlify.com/#/repos